python/src/HyperbolicAnisotropicEnumerateFunction_doc.i.in

```diff
@@ -35,14 +35,16 @@ effects, i.e. with a reasonably small number of nonzero components, prior to
 selecting those associated with higher-order interactions.
 
 For any real number :math:`q \in ]0, 1]`, one defines the anisotropic hyperbolic
-norm of a multi-index :math:`a` by:
+norm of a multi-index :math:`\vect{\alpha}` by:
 
 .. math::
     
-    \| \vect{a} \|_{\vect{w}, q} = \left( \sum_{i=1}^{n_X} w_i a^q \right)^{1/q}
+    \| \vect{\alpha} \|_{\vect{w}, q} = \left( \sum_{i=1}^{n_X} w_i \alpha_i^q \right)^{1/q}
 
-where the :math:`w_i`'s are real positive numbers. This would lead to first
-select the basis polynomials depending on a specific subset of input variables.
+where :math:`n_X` is the number of input variables and :math:`w_i > 0` are 
+real positive numbers called the weights. 
+This lead to select first the basis polynomials depending on a specific subset 
+of input variables.
```

Comment:
```suggestion
Monomials of input variables with larger weights are selected first
for the polynomial basis.
```

Reply:
It is the **axact opposite**!
```
import openturns as ot

enum_fun = ot.HyperbolicAnisotropicEnumerateFunction([1, 2, 4])
for i in range(20):
    print("i=", i, "enum=", enum_fun(i))
```
produces:
```
i= 0 enum= [0,0,0]
i= 1 enum= [1,0,0]
i= 2 enum= [0,1,0]
i= 3 enum= [2,0,0]
i= 4 enum= [3,0,0]
i= 5 enum= [0,0,1]
i= 6 enum= [0,2,0]
i= 7 enum= [4,0,0]
i= 8 enum= [5,0,0]
i= 9 enum= [0,3,0]
i= 10 enum= [6,0,0]
i= 11 enum= [7,0,0]
i= 12 enum= [0,0,2]
i= 13 enum= [0,4,0]
i= 14 enum= [8,0,0]
i= 15 enum= [1,1,0]
i= 16 enum= [9,0,0]
i= 17 enum= [0,5,0]
i= 18 enum= [10,0,0]
i= 19 enum= [11,0,0]
```
As you can see, high degrees of the first variable are selected first compared to the other variables!

Reply:
Oh, I got this completely mixed up. I guess this shows that the sentence needed to be made more precise. What do you think of this?
```suggestion
Monomials of input variables with smaller weights are selected first
for the polynomial basis.
```

Reply:
ok for me if you replace polynomial basis by functional basis and monomials by function ;-)! Monomial is misleading as e.g. Hermite polynomials are not monomials except for the degree 0, and the bases are not limited to polynomials. I use quite often the Haar basis:
```
enumFunction = ot.HyperbolicAnisotropicEnumerateFunction([1, 2, 4], 0.7)
basis = ot.OrthogonalProductFunctionFactory([ot.HaarWaveletFactory()]*3, enumFunction)
```

Reply:
```suggestion
Functions of input variables with smaller weights are selected first
for the functional basis.
```

Reply:
I suggest to include Regis's example in the doc, since no sentence can be as clear as the output of the function. But I admit that this is not as easy to make it compact...